_browser/blockstack_storage.md

```diff
@@ -578,8 +532,13 @@ process URL paths that contain the `.` (dot) character for example,
 3. Add a single method to the `Profile` class that determines if the app is viewing the local user's profile or another user's profile.
 
     ```javascript
-    isLocal() {
-      return this.props.match.params.username ? false : true
+    // Make sure you add this new prop!
+    export const Profile = ({ userData, handleSignOut, match }) => {
+      // ...
+      const isLocal = () => {
```

Comment:
where in the file should this go? the "..." before and after make it difficult to understand that